testing/internal/e2e/scope/scope_test.go

```diff
@@ -0,0 +1,322 @@
+package scope_test
```

Comment:
We'll have to think about file organization here. For now, I might suggest to add `scope_test.go` to the `static_test` package in `e2e/host/static` (you might not see this until you rebase with main). That way, it will automatically get run in the existing enos scenario. The alternative is to add another enos scenario, which would add some additional overhead in CI.

Reply:
I'll move it